### PR TITLE
Add basic docs for createTestEnv

### PR DESCRIPTION
**Pull request by ds300**
This was a little past-due 😬 

Realised when writing this that it doesn't support query variables yet 😞 I guess we never needed them in the order app. Should be pretty trivial to add when needed.

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 209591988e5143cbac27a7e79766bdbe97b2f5e0

https://deploy-preview-2253--artsy-reaction.netlify.com

**Comment by sweir27** [comment]
😍 Thank you!!

**Comment by artsyit** [comment]
:rocket: PR was released in v15.12.3 :rocket:

File: src/DevTools/README.md [hunk 1]
```diff
@@ -152,3 +153,148 @@ mockMutationResults = {
...
+```ts
+const page = await env.buildPage({
+  mockData: { /* different mock fixtures */ }
+  mockMutationResult: { /* different mutation mocks */ }
+})
+```
+
+### Dynamic things
+
+If you need to modify the the mutation results on-the-fly, you can use `env.mutations`
```
Comment (alloy) [inline src/DevTools/README.md]: > the the

File: src/DevTools/README.md [hunk 2]
```diff
@@ -152,3 +153,148 @@ mockMutationResults = {
...
+
+You can mock network failures
+
+```ts
+env.mutations.mockNetworkFailureOnce()
+await page.clickIncrement()
+expect(page.text()).toMatch("Netork error. Try again later.")
+await page.clickIncrement()
+expect(page.text()).not.toMatch("Netork error. Try again later.")
+```
```
Comment (alloy) [inline src/DevTools/README.md]: > Net**w**ork

File: src/DevTools/README.md [hunk 3]
```diff
@@ -152,3 +153,148 @@ mockMutationResults = {
...
+```ts
+expect(env.mutations.mockFetch).toHaveBeenCalledTimes(0)
+await page.clickIncrement()
+expect(env.mutations.mockFetch).toHaveBeenCalledTimes(1)
+```
+
+You can mock network failures
+
+```ts
+env.mutations.mockNetworkFailureOnce()
```
Comment (alloy) [inline src/DevTools/README.md]: Nice ✨

**Comment by damassi** [comment]
Nice ❇️

**Pull request merged**